src/main/webapp/app/entities/flair-bi/filter/filter-penal-pin.component.js

```diff
@@ -0,0 +1,177 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('flairbiApp')
+        .component('filterPenalPinComponent', {
+            templateUrl: 'app/entities/flair-bi/filter/filter-penal-pin.component.html',
+            controller: filterPenalPinController,
+            controllerAs: 'vm',
+            bindings: {
+                dimensions: '=',
+                view: '=',
+            }
+        });
+
+    filterPenalPinController.$inject = ['$scope', '$rootScope', 'filterParametersService', 'FilterStateManagerService', 'VisualDispatchService', 'SEPARATORS', '$stateParams', 'proxyGrpcService', 'favouriteFilterService'];
+
+    function filterPenalPinController($scope, $rootScope, filterParametersService, FilterStateManagerService, VisualDispatchService, SEPARATORS, $stateParams, proxyGrpcService, favouriteFilterService) {
+        var vm = this;
+
+        vm.load = load;
+        vm.applyFilter = applyFilter;
+        vm.list = {};
+        vm.selectedFilter = {};
+        activate();
+
+        ////////////////
+
+        function activate() {
+
+            vm.setting1 = {
+                scrollableHeight: '200px',
```

Comment:
rename to vm.setting1 to vm.settingStyle

Reply:
done